scripts/create-data.ts

```diff
@@ -0,0 +1,26 @@
+import * as faker from 'faker';
```

Comment:
Instead of create-data, please name this file create-pagelink-data or something like that